src/compaction_execute_bench.rs

```diff
@@ -231,10 +232,15 @@ impl CompactionExecuteBench {
             destination: 0,
             ssts,
             sorted_runs: vec![],
+            is_dest_last_run,
         })
     }
 
-    fn load_compaction_as_job(manifest: StoredManifest, compaction: &Compaction) -> CompactionJob {
+    fn load_compaction_as_job(
+        manifest: StoredManifest,
+        compaction: &Compaction,
+        is_dest_last_run: bool,
```

Comment:
can we rename to `skip_tombstones`? I'd rather the model be that the orchestrator just specify what it wants the executor to do than for the executor to decide whether to include tombstones or not based on whether or not it's the last sorted run.

Reply:
yes